diffsims/tests/test_utils/test_sim_utils.py

```diff
@@ -132,3 +145,176 @@ def test_kinematic_simulator_invalid_illumination():
 def test_uvtw_to_uvw(uvtw, uvw):
     val = uvtw_to_uvw(uvtw)
     np.testing.assert_almost_equal(val, uvw)
+
+
+class TestHolzCalibration:
+    def test_get_holz_angle(self):
+        wavelength = 2.51 / 1000
+        lattice_parameter = 0.3905 * 2 ** 0.5
+        angle = get_holz_angle(wavelength, lattice_parameter)
+        assert approx(95.37805 / 1000) == angle
+
+    def test_scattering_angle_to_lattice_parameter(self):
+        wavelength = 2.51 / 1000
+        angle = 95.37805 / 1000
+        lattice_size = scattering_angle_to_lattice_parameter(wavelength, angle)
+        assert approx(0.55225047) == lattice_size
+
```

Comment:
one nice additional test for functionality like this is to write a "loop" of tests. So `get_holz_angle()` and `scattering_angle_to_lattice_parameter()` are inverses to one another so a statement like:

`scattering_angle_to_lattice_parameter(wavelength,get_holz_angle(wavelength, lattice_parameter))`

should always return lattice_parameter independent of wavelength. You can then run over a range of wavelengths to check the functions still work (in this case it's fairly clear they will, but testing is for making sure that "fairly clear" stuff actually does work as you expect).

Reply:
Did you do something about this @isabelwood100 ?

Reply:
Yes i added in the test_lattice_paramter_invariance which should test this.